generators/app/prompts.js

```diff
@@ -9,6 +13,21 @@ module.exports = {
           name: 'seedbank',
           checked: true,
           value: "gem 'seedbank'"
+        },
+        {
+          name: 'rack-cors',
+          checked: true,
+          value: "gem 'rack-cors'"
+        },
+        {
+          name: 'redis',
+          checked: true,
+          value: "gem 'redis', '~> 3.0'"
+        },
+        {
+          name: 'bcrypt',
+          checked: true,
+          value: "gem 'bcrypt', '~> 3.1.7'"
```

Comment:
I agree 100% on `minitest` as the default, but not on the default `minitest` settings being the default.

I also don't think it will be very hard to add RSpec as an option. Yeoman is made exactly for those kind of scenarios, it wouldn't be really like maintaining two versions of the template app.